### ISSUE TITLE
508-defect-2: Headings in the summary and service verification letter accordion should be logically ordered

### ISSUE THREAD

**Issue by sortizsh** [department-of-veterans-affairs/va.gov-team#45944]
## Point of contact

Josh Kim

## List of issues by severity
There are 3 tickets related to this issue: 
Main ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45650
Heading: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45944
Labelling: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45945
Responsive: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45946
Download: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45998
Visual Instructions: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45999
I've grouped these issues together as they're closely related and dependent on one another. Whatever solution we consider will likely need to take into account all of these issues together from a holistic perspective @skylerschain 

### Headings aren't logically ordered (Major)
"Choose the information..." is a `h3` despite its parent being a `h3`; depending on intent, this may not be a heading at all.

<img width="1537" alt="headings" src="https://user-images.githubusercontent.com/14154792/184410904-effee184-6438-4033-ba4a-ba6a62bc5e82.png">

## Reproduction steps

1. [Log in as user 299 and get to step 2](https://staging.va.gov/records/download-va-letters/letters/letter-list)
2. Open the benefit summary accordion

## Proposed solution or next steps
- remove h3 from "choose the information you want to include" and make paragraph text

## Type of issue

- [x] Screenreader
- [x] Keyboard
- [ ] Focus
- [x] Headings
- [ ] Color
- [x] Zoom
- [x] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by sortizsh** [comment]
Pulled from this larger ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45650

**Comment by sortizsh** [comment]
@jacobworrell  this one can be prio'd

**Comment by joshkimux** [comment]
Small bump for you @jacobworrell just in case it was lost ❤️
